src/codegen/generate.test.ts

```diff
@@ -55,6 +55,15 @@ describe("generate", () => {
 
     expect(oneLine).toContain(circleTypeDefinition);
   });
+
+  it("should handle enums", async () => {
+    const api = printAst(new ApiGenerator(spec.petstore).generateApi());
+    const oneLine = api.replace(/\s+/g, " ");
+
+    const enumTypeDefinition = `export type Option = ("one" | "two" | "three")[];`;
+
+    expect(oneLine).toContain(enumTypeDefinition);
+  });
```

Comment:
This tests as-is behavior, right?

Reply:
Yes exactly. I moved around some of the enum processing code so wanted to have a little test coverage there to make sure I didn't break existing behavior. I would have used the existing pet.json example for the "const" test, but since "const" is technically OAS 3.1, I didn't want to add anything out of spec to that file. If more 3.1 features make it in, it will probably make sense to keep a pet-3.1.json example file around that future tests can go off of.

Reply:
Cool! Thanks for elaborating and for taking the extra afford 👍